news/vulnerabilities.xml

```diff
@@ -1594,6 +1594,53 @@ issue.
     <advisory url="/news/secadv/20170126.txt"/>
     <reported source="OSS-Fuzz project" />
   </issue>
+
+  <issue public="20160824">
+    <impact severity="Low"/>
+    <cve name="2016-2183"/>
+    <affects base="1.0.2" version="1.0.2"/>
+    <affects base="1.0.2" version="1.0.2a"/>
+    <affects base="1.0.2" version="1.0.2b"/>
+    <affects base="1.0.2" version="1.0.2c"/>
+    <affects base="1.0.2" version="1.0.2d"/>
+    <affects base="1.0.2" version="1.0.2e"/>
+    <affects base="1.0.2" version="1.0.2f"/>
+    <affects base="1.0.2" version="1.0.2g"/>
+    <affects base="1.0.2" version="1.0.2h"/>
+    <fixed base="1.0.2" version="1.0.2i" date="20160922"/>    
```

Comment:
Is it worth noting that 1.1.0 also fixed this?
`<fixed base="1.1.0" version="1.1.0" date="20160825"/>`